plugin.xml

```diff
@@ -3,5 +3,5 @@
   <feature extension="package.version" value="2.0.0"/>
   <transtype name="org.dita-ot.html"/>
   <feature extension="dita.conductor.target.relative" file="integrator.xml"/>
-  <feature extension="dita.conductor.html5.param" file="params.xml"/>
+  <feature extension="dita.conductor.html.param" file="params.xml"/>
```

Comment:
Latest commit in master reverts this, but I'm curious what this break in 2.3? I thought the difference (besides html5 not working in 2.2) was only semantic.